### PR TITLE
Contact Form : Validations and sending email

### PR DESCRIPTION
**Pull request by meriouma**

### CODE REVIEW

**Comment by meriouma** [comment]
@LarryMatte Did you just close your PR or you merged it? It seems after updating with master I still have your changes

File: src/main/resources/com/arcbees/website/client/resources/css/widget/contactform.gss [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+    @mixin animation_name(sendIcon);
+    @mixin animation_duration(2s);
+    @mixin animation_timing_function(linear);
+    @mixin animation_fill_mode(fowards);
+
+}
+
+@keyframes sendIcon {
+    0% {
+        -moz-transform:rotate(0deg) translate(0, 0);
```
Comment (christiangoudreau) [inline src/main/resources/com/arcbees/website/client/resources/css/widget/contactform.gss]: Why can't we create a mixin for that?

Comment (jasonlemay) [inline src/main/resources/com/arcbees/website/client/resources/css/widget/contactform.gss]: We can : `@mixin transform(rotate(0deg) translate(0, 0))`

File: src/main/resources/com/arcbees/website/client/resources/css/widget/contactform.gss [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+    60% { -webkit-transform: translate(-3px, 1px) rotate(0deg); }
+    70% { -webkit-transform: translate(2px, 1px) rotate(-1deg); }
+    80% { -webkit-transform: translate(-1px, -1px) rotate(1deg); }
+    90% { -webkit-transform: translate(2px, 2px) rotate(0deg); }
+    100% { -webkit-transform: translate(1px, -2px) rotate(-1deg); }
+}
+
+@keyframes error {
+    0% {
+        -moz-transform: translate(2px, 1px) rotate(0deg);
```
Comment (jasonlemay) [inline src/main/resources/com/arcbees/website/client/resources/css/widget/contactform.gss]: @meriouma you can use the transform mixin here too

**Pull request merged**